db/migrate/20250627084852_not_null_organization_id_on_applied_usage_thresholds.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class NotNullOrganizationIdOnAppliedUsageThresholds < ActiveRecord::Migration[8.0]
+  def up
+    validate_check_constraint :applied_usage_thresholds, name: "applied_usage_thresholds_organization_id_null"
+    change_column_null :applied_usage_thresholds, :organization_id, false
```

Comment:
For my knowledge, couldn't we change the column to NOT NULL without validating the check first ? Wouldn't PG validate the constraints against the table anyway?

Reply:
Interesting. I couldn't find the PG doc related to this behaviour. So I tested it locally and indeed, it does not lock 🙌